spec/features/loan_details_spec.rb

```diff
@@ -1,48 +1,73 @@
-require 'rails_helper'
+require "rails_helper"
 
-feature 'loan details' do
-  before { pending 're-implement in new project' }
-  let(:loan) { create(:loan) }
-  before { visit loan_path(loan) }
+feature "loan details" do
+  let!(:div_us) { create(:division, name: "United States", short_name: "us") }
+  let!(:loan) do
+    create(
+      :loan,
+      :public, :active, :with_translations, :with_loan_media, :with_coop_media, :with_log_media,
+      division: div_us
+    )
+  end
+  let!(:organization) { loan.organization }
+  let!(:active_loans) do
+    create_list(:loan, 2,
```

Comment:
Does the 2 in this line mean that two active loans are created?

Reply:
Yes.